### PR TITLE
Unit test of Kubejobs components and documentation

### PR DESCRIPTION
**Pull request by rafaelvfalc**
- Addition of Unit tests of the Kubejobs component
- Documentation about how run tests

### CODE REVIEW

File: broker/tests/unit/plugin/kubejobs/test_plugin.py [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+import threading
+
+from broker.plugins.kubejobs.plugin import KubeJobsExecutor
+from broker.plugins.kubejobs.plugin import KubeJobsProvider
+
+from broker.tests.unit.mocks.k8s_mock import MockKube
+from broker.tests.unit.mocks.redis_mock import MockRedis
+
+
+class TestKubeJobsPlugin(unittest.TestCase):
```
Comment (GabrielSVinha) [inline broker/tests/unit/plugin/kubejobs/test_plugin.py]: Same.l

File: broker/tests/unit/mocks/redis_mock.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class MockRedis():
```
Comment (GabrielSVinha) [inline broker/tests/unit/mocks/redis_mock.py]: Please add class/functions documentation.

File: broker/tests/unit/mocks/k8s_mock.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+class Job():
+   def __init__(self, active):
+       self.status = Status(active)
+
+class Status():
```
Comment (GabrielSVinha) [inline broker/tests/unit/mocks/k8s_mock.py]: Example of documentation:

Comment (GabrielSVinha) [reply broker/tests/unit/mocks/k8s_mock.py]:
```
def something(p1=None):
    """
    Do something
    :param p1: first parameter
    :raises AttributeError: When something happens
    """
```

File: broker/tests/unit/plugin/kubejobs/test_plugin.py (hunk 1 above)
Comment (rafaelvfalc) [reply broker/tests/unit/plugin/kubejobs/test_plugin.py]: Done!

Comment (rafaelvfalc) [reply broker/tests/unit/plugin/kubejobs/test_plugin.py]: Done!

File: broker/tests/unit/mocks/redis_mock.py (hunk 2 above)
Comment (rafaelvfalc) [reply broker/tests/unit/mocks/redis_mock.py]: Done!

File: broker/tests/unit/mocks/k8s_mock.py (hunk 3 above)
Comment (rafaelvfalc) [reply broker/tests/unit/mocks/k8s_mock.py]: Done!

**Pull request merged**